saleor/graphql/account/mutations/account.py

```diff
@@ -28,6 +29,12 @@
 class AccountRegisterInput(graphene.InputObjectType):
     email = graphene.String(description="The email address of the user.", required=True)
     password = graphene.String(description="Password.", required=True)
+    redirect_origin = graphene.String(
```

Comment:
In other mutations where this pattern is used this argument is called `redirectUrl` - for consistency, I would use it here as well.